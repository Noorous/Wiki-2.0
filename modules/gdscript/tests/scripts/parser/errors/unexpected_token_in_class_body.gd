# Fixes github.com/godotengine/godot/issues/96792

var test = true
error = true

func test():
	pass
