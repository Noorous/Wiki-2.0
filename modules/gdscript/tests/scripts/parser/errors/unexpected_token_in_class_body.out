GDTEST_PARSER_ERROR
Unexpected "error" in class body.
